istio.deps

```diff
@@ -11,6 +11,6 @@
     "name": "ZTUNNEL_REPO_SHA",
     "repoName": "ztunnel",
     "file": "",
-    "lastStableSHA": "cdd8c07aea7f9e991b6eeb16d6444459f79d5a02"
+    "lastStableSHA": "ea748c65254a64dcb644f2b3aae54ed05c3ea53e"
```

Comment:
NOTE: will not merge with this. Just changing it to run the ambient test against the latest ztunnel, since the ztunnel update is currently blocked by an unrelated issue.

Reply:
Confirmed ambient tests pass with the updated ztunnel binary. Rolling back this change.